doc/cookbook/source/examples/classifier/mkl_binary_class.rst

```diff
@@ -0,0 +1,55 @@
+==================================================
+Multiple Kernel Learning for Binary Classification
+==================================================
+
+Multiple kernel learning (MKL) is based on convex combinations of arbitrary kernels over different domains. One could make a combination of kernels like:
+
+.. math::
+
```

Comment:
I wonder whether we should say one sentence about how the MKL solver works here.